src/lib.rs

```diff
@@ -61,20 +61,15 @@ pub fn resolve(args: ClapperArgs) -> Result<(), Box<dyn Error>> {
         },
         EntityType::Scaffold(scaffold_command) => {
             match scaffold_command.command {
-                ScaffoldSubCommand::Reactjs(dir) => {
-                    match create_react_app(dir.dir_name.clone()) {
+                ScaffoldSubCommand::React(dir) => {
+                    let result = match collect_arguement_from_the_terminal().trim() {
+                        "j" => create_react_app(dir.dir_name.clone()),
+                        "t" => create_react_app_with_typescript(dir.dir_name.clone()),
```

Comment:
Similarly the `create_react_app_with_typescript()` returns a Result enum, handle all variants.